Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.8.24 (unreleased)
 
 Language Features:
-
+ * Introduce global ``blobhash`` for retrieving the versioned blob hash of a given block.
```

Comment:
Cancun related changes should also be listed under [Target Options](https://docs.soliditylang.org/en/latest/using-the-compiler.html#target-options) in the docs.